File: src/main/java/fr/vlegall/sochief/repository/IngredientRepository.java
package fr.vlegall.sochief.repository;

import fr.vlegall.sochief.model.recipe.Ingredient;
import org.springframework.data.jpa.repository.JpaRepository;

public interface IngredientRepository extends JpaRepository<Ingredient, Long> {
    Ingredient findByNameIgnoreCase(String name);
}

File: src/main/kotlin/fr/vlegall/sochief/repository/IngredientRepository.kt
package fr.vlegall.sochief.repository

import fr.vlegall.sochief.model.recipe.Ingredient
import org.springframework.data.jpa.repository.JpaRepository

interface IngredientRepository : JpaRepository<Ingredient, Long> {
    fun findByNameIgnoreCase(name: String): Ingredient?
}
